pkg/cmd/ui/ui.go

```diff
@@ -282,3 +282,16 @@ func SelectOptionPushToGit() bool {
 	handleError(err)
 	return optionPushToGit == "yes"
 }
+
+// SelectOptionAccessToken allows users the option to pass the access token to set up automated pushes and also commit-status-tracker
```

Comment:
Again, how about `UsePersonalAccessToken` ?

I'm not sure about `git personal access token`, but I'm not sure what's better than that